src/current/_includes/common/telemetry-table.html

```diff
@@ -0,0 +1,932 @@
+      "leasePreferences": [
+      ],
+      "inheritedLeasePreferences": false,
+      "subzones": [
+        {
+          "indexId": 1,
+          "partitionName": "f3f81b68",
+          "config": {
+            "rangeMinBytes": null,
+            "rangeMaxBytes": null,
+            "gc": null,
+            "globalReads": null,
+            "numReplicas": null,
+            "numVoters": null,
+            "constraints": [
+              {
+                "numReplicas": 0,
+                "constraints": [
+                  {
+                    "type": 1,
+                    "key": "2cd13cd0",
+                    "value": "d006156b"
+                  }
+                ]
+              }
+            ],
+            "voterConstraints": [
+            ],
+            "inheritedConstraints": false,
+            "nullVoterConstraintsIsEmpty": false,
+            "leasePreferences": [
+            ],
+            "inheritedLeasePreferences": true,
+            "subzones": [
+            ],
+            "subzoneSpans": [
+            ]
+          }
+        }
+      ],
+      "subzoneSpans": [
+      ]
+    }
+    }
+  </pre>
+  </td>
+ </tr>
+ <tr>
+  <td>Feature usage</td>
+  <td>List of feature names and counters showing how often the features are used.</td>
+  <td>
+   <ul>
+    <li>Feature Name</li>
+    <li>Usage Count</li>
+   </ul>
+  </td>
+  <td>
+   <pre>
+"featureUsage": {
+        "backup.duration-sec.inc-succeeded.10": 10,
+        "sql.plan.builtins.gen_random_uuid() -> uuid": 100000,
+        "sql.plan.ops.cast.float::string": 100000,
+        "sql.query.count": 1000000000,
+        "sql.txn.commit.count": 100000000,
+        "txn.restarts.serializable": 10000,
+        "backup.speed-mbps.inc.per-node.100": 100,
+        "external-io.google_cloud": 10000
+    }
+  </pre>
+  </td>
+ </tr>
+ <tr>
+  <td>Locality</td>
+  <td>
+   <p>
+A locality is an ordered set of key/value pairs that describe the location of this node or SQL instance. This is the locality information that is passed at node start time using the <a href="cockroach-start.html#locality">
+     <code>cockroach start --locality</code>
+    </a> command.
+</p>
+   <p>
+Note that these values are not redacted. Avoid writing information about your users or customers using the <pre>--locality</pre> flag.
+</p>
+  </td>
+  <td>
+   <ul>
+    <li>Tiers</li>
+   </ul>
+  </td>
+  <td>
+   <pre>
+ "locality": {
+      "tiers": [
+        {
+          "key": "a5a7e0e9",
+          "value": "ded62d43"
+        }
+      ]
+    }
+  </pre>
+  </td>
+ </tr>
+ <tr>
+  <td>SQL statistics</td>
+  <td>SQL statement statistics.
+
+Note that table names, column names, and constants are redacted. The <pre>sensitiveInfo</pre> field in the output is also redacted, since it can contain error strings from SQL execution. The <pre>app</pre> field is also redacted unless it begins with <pre>$</pre> or <pre>$$</pre> which is used by CockroachDB internally.
```

Comment:
Ditto on pre tags

Reply:
have updated this and all other `pre` tags to use `code`